### DISCUSSION TITLE
Replace '-' character in playlist leaderboard with last time played

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19291] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/74088315/180353862-eb30cdee-7baf-481f-8319-a69e5166e236.png)

So currently in playlists, when you hover over the '-' character next to a user's country flag, it will display a date of 1 January 0001. I'm assuming that this is a placeholder for now, but I think that it would be cool if the '-' character was replaced with the last time the user played in this specific playlist so that you can get a better idea of how recently someone played in a playlist. I'm not sure if this is possible to do just yet but it's something that I think would be nice to see in the future.

**Comment by peppy** [comment, marked as answer]
Seems like an oversight with the data not being mapped correctly from the request.

**Reply by frenzibyte** [reply]
Escalated to #19292
